src/Api/User/editUser/index.js

```diff
@@ -1,31 +1,30 @@
+import { getUserId } from '../getUserId';
+import useRemoveToken from '../../removeToken';
 import axios from '../../../axiosConfig'
 import { enqueueSnackbar } from 'notistack'
 import { useCookies } from 'react-cookie'
-import { ROUTES } from '../../../constants';
-import { getUser } from '../getUser';
 
 const useEditUser = () => {
-    const [cookies, setCookies] = useCookies(["access_token"]);
-    const user = getUser();
-  const editUser = async (username, password) => {
+  const [cookies, setCookies] = useCookies(["access_token"]);
+  const { checkIfInvalidToken } = useRemoveToken();
+    const userId = getUserId();
+  const axiosEditUser = async (username, password) => {
     try {
-    const res = await axios.put(`/users/editUser/${user._id}`, {
+    const res = await axios.put(`/auth/editUser/${userId}`, {
       username,
```

Comment:
Extract the request to your server into a different service and from the service call this funtions 

You can have a class or a file that contains all the requests, for example :


`LoginServise.doSomething()`

Reply:
i dont understand how it should look like it new to me